src/index-generator.ts

```diff
@@ -0,0 +1,263 @@
+import * as fs from 'fs';
```

Comment:
Do we have any copyright information for this?

Reply:
done.